tests/api/routers/test_filing_api.py

```diff
@@ -1027,6 +1027,9 @@ async def test_good_sign_filing(
             action_type=UserActionType.SIGN,
         )
 
+        send_email_mock = mocker.patch("sbl_filing_api.routers.filing.send_confirmation_email")
+        send_email_mock.return_value = None
+
         upsert_mock = mocker.patch("sbl_filing_api.entities.repos.submission_repo.upsert_filing")
         updated_filing_obj = deepcopy(get_filing_mock.return_value)
         upsert_mock.return_value = updated_filing_obj
```

Comment:
You should use the send_email_mock to verify that the correct data is being send to that function, to test that the /sign endpoint is correctly passing along the user name, email, contact info email, confirmation id, and timestamp.